pkg/cmd/build/log_filter.go

```diff
@@ -29,17 +29,17 @@ type Rule struct {
 	transformer TransformerFunc
 }
 
-type BuildKitLogsFilter struct {
+type LogsFilter struct {
```

Comment:
I have doubts about this one. `BuildKit` is not the same than `build`. I guess this is detecting it because `BuildKit` is in camelcase. If we use `BuildkitLogsFilter` it might pass the validation.

I have doubts because this seems specific to BuildKit and not to build. If we do builds with other product (like Depot), we might need to use other filter. In this case, the prefix could be relevant. WDYT?

Reply:
good call, changed in c7ba65fb